k8s/charts/openebs/values.yaml

```diff
@@ -26,15 +26,28 @@ apiserver:
 
 provisioner:
   image: "openebs/openebs-k8s-provisioner"
-  imageTag: "0.5.4"
+  imageTag: "0.6.0-RC1"
   replicas: 1
   nodeSelector: {}
   tolerations: {}
   affinity: {}
 
+snapshotOperator:
+  controller:
+    image: "openebs/snapshot-controller"
+    imageTag: "0.6.0-RC1"
+  provisioner:
+    image: "openebs/snapshot-provisioner"
+    imageTag: "0.6.0-RC1"
```

Comment:
RC2 ?

Reply:
Instead of "snapshot-controller", it should be  "snapshot-operator"?